analysis_configs/met_filters.py

```diff
@@ -1,4 +1,4 @@
-met_filters = [
+met_filters_treemaker = [
```

Comment:
The TreeMaker configs needs to be updated

Reply:
done